### DISCUSSION TITLE
Github Webhook Not Working with Jenkins under Teleport.

### DISCUSSION THREAD

**Discussion by Unthink-pri18** [gravitational/teleport#49181] [Q&A]
I have Jenkins Setup(using App Service) under Teleport SSO and JWT authorizers for the Auth management, When I am trying to setup the Webhook Triggers for a pipeline in Github,the payload is giving 302 error code.

**Comment by webvictim** [comment]
Teleport doesn't support unauthenticated access to any app endpoints which are hosted behind it - this includes webhooks.

You will likely need to configure some other reverse proxy to allow access to the webhook URL. My recommendation is a web server like nginx or Caddy which only allows connections to the webhook URL, and restricts this access to Github IP ranges only.
